eahub/base/static/scripts/main.js

```diff
@@ -41,3 +41,15 @@ var navbar = document.getElementById('navbar')
 menu_btn.addEventListener('click', function() {
   navbar.style.display = navbar.style.display == 'inline-block' ? 'none' : 'inline-block';
 })
+
+var claim_group = document.getElementById('claim_group')
```

Comment:
Which page(s) does this code run on? If it's just on the details page, then I'm confused about why we don't just control this server-side.

Reply:
Not quite sure what you mean. This runs on group.html and allows toggling the display of the info field about group claims (in interaction with the bootstrap functionality for closing that field)